codestodians/food-basket-api
# [ft: #154437201] Allow Travis handle Heroku Deployment

#### What does this PR do?
- It activates `Travis CI` as the handler for heroku deployment

#### Description of Task to be completed?
- Add the config options Travis requires to handle automated deployment

#### How should this be manually tested?
- Check the Travis Build for the push. It can be observed that this particular branch was deployed when added to the list of permitted branches.

#### What are the relevant pivotal tracker stories?
- [#1542067020](https://www.pivotaltracker.com/story/show/1542067020)

#### Screenshots:
<img width="1280" alt="screen shot 2018-01-18 at 6 12 13 pm" src="https://user-images.githubusercontent.com/22825700/35111325-518e5974-fc7b-11e7-8029-e35e5e16c2ab.png">
